external/src/main/java/org/hawkular/alerts/AlertsStandalone.java

```diff
@@ -65,14 +66,14 @@
     private ExtensionsServiceImpl extensions;
     private IncomingDataManagerImpl incoming;
     private IspnActionsServiceImpl ispnActions;
-    private IspnAlertsServiceImpl ispnAlerts;
     private IspnDefinitionsServiceImpl ispnDefinitions;
     private StatusServiceImpl status;
     private PartitionManagerImpl partitionManager;
     private PublishCacheManager publishCacheManager;
     private IspnAdminService adminService;
 
-    public AlertsStandalone() {
+    @PostConstruct
+    void postConstruct() {
         log.info("Policies Engine uses Infinispan backend");
```

Comment:
Is that still true? See L107

Reply:
Depending on the `policies-history.alerts-service.enabled` conf value, it can be completely true or partly true 😃